php/Terminus/Collections/UserMemberships.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace Terminus\Collections;
+use Terminus\Request;
+use Terminus\Session;
+use Terminus\UserMembership;
+use \Terminus_Command;
+
+class UserMemberships {
```

Comment:
This may be a painful change, but can you please update this to be `SiteUserMemberships`. Cause OrganizationUserMemberships are slightly different :cry: